addon/utils/text.js

```diff
@@ -71,8 +72,9 @@ export function trimDataToFit (text, width, font) {
  * Note: expects data-text attribute to be on element with full text
  *
  * @param {HTMLElement} element - HTML element to trim data within
+ * @param {jQuery} [$] - jQuery selector, can be stubbed for testing
```

Comment:
I guess it's just that the parameter becomes overwrites the `$` from outside it's scope before it can be set to the default value. Renaming the `$` also works: `$2 = $`.